### ISSUE TITLE
label and status should be clickable

### ISSUE THREAD

**Issue by karlcow** [webcompat/webcompat.com#2104]
When going to an issue page. It should be possible to click on the label and status and go to the relevant search page.

**Comment by miketaylr** [comment]
Makes sense, this is a useful behavior that GitHub has.

**Comment by zoepage** [comment]
@karlcow You mean in the issue detail page as it works in the issue list?

**Comment by karlcow** [comment]
yes, but not a comment part of the refactor. This is a bug which exists on the current design.
<img width="308" alt="capture d ecran 2018-02-19 a 21 09 04" src="https://user-images.githubusercontent.com/505230/36377117-295bda56-15b9-11e8-8bb4-0b36d2b85c7d.png">

**Comment by zoepage** [comment]
I think, this is a good idea and we should add this feature.

So, the click on the label would direct to the `issue-list` with search params for the label or milestone.

**Comment by Obtin** [comment]
@miketaylr @zoepage I would like to work on this issue.

**Comment by karlcow** [comment]
feel free to tackle it after #2380
Thanks a lot. That's cool!
